core/src/com/biglybt/core/util/UrlUtils.java

```diff
@@ -1254,36 +1251,31 @@ public static String unescapeXML(String s) {
 
 		Map	result = new HashMap();
 
-		try{
```

Comment:
Variable `headers_to_use` is always none null, so no exceptions from this block.